esmvalcore/_task.py

```diff
@@ -642,8 +642,8 @@ def __repr__(self):
         settings_string = pprint.pformat(self.settings)
         string = (f"{self.__class__.__name__}: {self.name}\n"
                   f"script: {self.script}\n"
-                  f"settings:\n{settings_string}\n")
-
+                  f"settings:\n{settings_string}\n"
+                  f"{super().str()}\n")
```

Comment:
Can we change this to something that does not use super?

```suggestion
                  f"{self.str()}\n")
```

Reply:
Good point, I've changed it. Several revisions ago the `str` method was called `__str__` so back then it made sense to use `super`.